spec/ocnless_cluster_spec.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "cluster"
+require "clustering/cluster_ht_item"
+
+RSpec.describe OCNLessCluster do
+  let(:bib_key1) { 1 }
+  let(:c1) { described_class.new(bib_key: bib_key1) }
+  let(:ht) { build(:ht_item).to_hash }
+
+  include_context "with cluster ocns table"
+
+  describe "#initialize" do
+    it "creates a new cluster" do
+      expect(c1.class).to eq(described_class)
+    end
+
+    it "has an ocns field that is an empty Set" do
+      expect(described_class.new(bib_key: bib_key1).ocns.class).to eq(Set)
+      expect(described_class.new(bib_key: bib_key1).ocns.size).to eq(0)
+    end
+  end
+
+  describe "#ht_items" do
+    include_context "with hathifiles table"
+
+    it "in a cluster with one bib_key, returns matching htitem" do
+      htitem = build(
+        :ht_item,
+        ocns: [],
+        ht_bib_key: bib_key1,
+        access: "allow",
+        rights: "pd",
+        collection_code: "PU"
+      )
+      insert_htitem(htitem)
+      cluster_items = c1.ht_items
+      expect(cluster_items.to_a.length).to eq(1)
+      expect(cluster_items.first.item_id).to eq(htitem.item_id)
+    end
+  end
+
+  describe "#holdings" do
+    it "returns an empty Array" do
+      expect(c1.holdings).to eq([])
+    end
+  end
+
+  describe "#format" do
+    it "has a format" do
+      formats = ["spm", "mpm", "ser"]
+      expect(formats).to include(c1.format)
+    end
+  end
+
+  # Not clear yet how to set up the scaffolding to test the methods we want to keep
```

Comment:
I'm not sure here either and we haven't re-enabled these specs for Cluster yet I don't think. I think it would be fine to note this that there may be future work here for the "production overlap table"